entries/s/split.io.json

```
@@ -0,0 +1,12 @@
+{
+  "Split": {
+    "domain": "split.io",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.split.io/hc/sections/12620642891149",
```

As the section only contains the following article, I think it makes more sense to link to the article itself.
```suggestion
    "documentation": "https://help.split.io/hc/articles/360020825212",
```

---

Thanks for your correction!